Assets/Scripts/SS3D/Systems/Tile/TileMapCreator/AssetGrid.cs

```diff
@@ -66,11 +66,10 @@ private void HandleUpdate(ref EventContext context, in UpdateEvent updateEvent)
     /// </summary>
     public void FindAssets(string text)
     {
-        text = text.Replace(' ', '_');
```

Comment:
why removing this line ?

Reply:
prefab name doesn't contains `_` 🤔

Reply:
@jckling that's right, I'm not sure why it was there in the first place to be honest. All our prefabs should use PascalCasing.

Reply:
it's because previous `nameString` is set in scriptable object, some have `_` and not same as prefab name